src/Raven.Client/Documents/Commands/PutDocumentCommand.cs

```diff
@@ -13,17 +13,17 @@ public class PutDocumentCommand : RavenCommand<PutResult>
         private readonly string _id;
         private readonly string _changeVector;
         private readonly BlittableJsonReaderObject _document;
-        private readonly JsonOperationContext _context;
 
         public PutDocumentCommand(string id, string changeVector, BlittableJsonReaderObject document, JsonOperationContext context)
         {
             _id = id ?? throw new ArgumentNullException(nameof(id));
             _changeVector = changeVector;
             _document = document ?? throw new ArgumentNullException(nameof(document));
-            _context = context ?? throw new ArgumentNullException(nameof(context));
+            if (context == null)
+                throw new ArgumentNullException(nameof(context));
```

Comment:
If you aren't using this, why are you checking it and actually getting it passed on?